src/main/java/slimeknights/tconstruct/gadgets/item/slimesling/BlueSlimeSlingItem.java

```diff
@@ -0,0 +1,52 @@
+package slimeknights.tconstruct.gadgets.item.slimesling;
+
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.entity.player.ServerPlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.world.World;
+import slimeknights.tconstruct.common.Sounds;
+import slimeknights.tconstruct.library.SlimeBounceHandler;
+import slimeknights.tconstruct.library.network.TinkerNetwork;
+import slimeknights.tconstruct.tools.common.network.EntityMovementChangePacket;
+
+public class BlueSlimeSlingItem extends BaseSlimeSlingItem {
+
+  public BlueSlimeSlingItem(Properties props) {
+    super(props);
+  }
+
+  /** Called when the player stops using an Item (stops holding the right mouse button). */
+  @Override
+  public void onPlayerStoppedUsing(ItemStack stack, World worldIn, LivingEntity entityLiving, int timeLeft) {
+    if (!(entityLiving instanceof PlayerEntity)) {
+      return;
+    }
+
+    PlayerEntity player = (PlayerEntity) entityLiving;
+    if (player.isElytraFlying()) {
```

Comment:
Add a comment here explaining why we disallow elytra flying. Loosely, the problem is slimeslings behave similarly to fireworks, so this is essentially a cheap version of fireworks.